contents/docs/error-tracking/cutting-costs.mdx

```diff
@@ -24,6 +24,35 @@ Alternatively, you can disable exception autocapture completely in your [project
 
 ## Filtering exceptions clientside
 
+### Suppression rules
+
+Autocaptured exceptions can be ignored client-side by [configuring suppression rules](https://us.posthog.com/error_tracking/configuration?tab=error-tracking-exception-autocapture#selectedSetting=error-tracking-exception-autocapture).
+Given the stack of an exception may still be minified client-side you are only permitted to filter based on the exception type and message attributes.
```

Comment:
```suggestion
Autocaptured exceptions can be ignored client-side by [configuring suppression rules](https://us.posthog.com/error_tracking/configuration?tab=error-tracking-exception-autocapture#selectedSetting=error-tracking-exception-autocapture). Because the stack of an exception may still be minified client-side, you can only filter based on the exception type and message attributes.
```